File: src/main/java/preponderous/viron/models/Location.java
// Copyright (c) 2024 Preponderous Software
// MIT License

package preponderous.viron.models;

public class Location {
    private int locationId;
    private int x;
    private int y;

    public Location(int locationId, int x, int y) {
        this.locationId = locationId;
        this.x = x;
        this.y = y;
    }

    public int getLocationId() {
        return locationId;
    }

    public void setLocationId(int locationId) {
        this.locationId = locationId;
    }

    public int getX() {
        return x;
    }

    public void setX(int x) {
        this.x = x;
    }

    public int getY() {
        return y;
    }

    public void setY(int y) {
        this.y = y;
    }

    @Override
    public String toString() {
        return "Location{" +
                "locationId=" + locationId +
                ", x=" + x +
                ", y=" + y +
                '}';
    }
}

File: src/main/python/preponderous/viron/models/location.py
# Copyright (c) 2024 Preponderous Software
# MIT License

class Location:
    def __init__(self, location_id: int, x: int, y: int):
        self.location_id = location_id
        self.x = x
        self.y = y

    def get_location_id(self) -> int:
        return self.location_id

    def set_location_id(self, location_id: int):
        self.location_id = location_id

    def get_x(self) -> int:
        return self.x

    def set_x(self, x: int):
        self.x = x

    def get_y(self) -> int:
        return self.y

    def set_y(self, y: int):
        self.y = y

    def __str__(self) -> str:
        return f"Location{{location_id={self.location_id}, x={self.x}, y={self.y}}}"